rust/android/Dockerfile

```diff
@@ -78,7 +79,7 @@ RUN cd /opt/openssl-3.0.1 && \
 
 RUN rustup component add rustfmt
 
-RUN cd /opt && git clone --depth=1 https://github.com/zecwalletco/zecwallet-mobile zecwalletmobile && cd zecwalletmobile/rust/android && cargo fetch && cargo build --release && rm -rf /opt/zecwalletmobile
+RUN cd /opt && git clone --depth=1 https://github.com/zingolabs/zecwallet-mobile zecwalletmobile && cd zecwalletmobile/rust/android && cargo fetch && cargo build --release && rm -rf /opt/zecwalletmobile
```

Comment:
I want to point out this change - the build still works with changing over to `zingolabs` for `zecwallet-mobile`! However, we are still pulling in `librustzcash` from `https://github.com/adityapk00/librustzcash` :
``` 
Step 20/39 : RUN cd /opt && git clone --depth=1 https://github.com/zingolabs/zecwallet-mobile zecwalletmobile && cd zecwalletmobile/rust/android && cargo fetch && cargo build --release && rm -rf /opt/zecwalletmobile
     ---> Running in 229dc95e960e
    Cloning into 'zecwalletmobile'...
    Updating git repository `https://github.com/zingolabs/zecwallet-light-cli`
    Updating git repository `https://github.com/adityapk00/librustzcash`
```

Reply:
https://github.com/zingolabs/librustzcash is currently "[26 commits behind](https://github.com/zingolabs/librustzcash/compare/master...zcash:master) zcash:master." and because aditya's https://github.com/adityapk00/librustzcash is "[68 commits ahead](https://github.com/adityapk00/librustzcash/compare/zcash:master...master), [796 commits behind](https://github.com/adityapk00/librustzcash/compare/master...zcash:master) zcash:master." I think this concern should be minimally separated out into another PR, though it would be interesting to try to just swap them out and see what happens!

Reply:
@AloeareV is working on this.